sale_fixed_discount/__manifest__.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2017-20 ForgeFlow S.L.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    "name": "Sale Fixed Discount",
+    "summary": "Allows to apply fixed amount discounts in sales orders.",
+    "version": "16.0.1.0.0",
+    "category": "Sales",
+    "website": "https://github.com/OCA/sale-workflow",
+    "author": "ForgeFlow, Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "application": False,
+    "installable": True,
+    "depends": ["sale", "account_invoice_fixed_discount"],
```

Comment:
Hi, this migration is linked to another pull request, you should make a link to this request by adding it in the requirement.txt file.  You can take a look at this documentation https://github.com/OCA/maintainer-tools/wiki/Use-temporary-reference%28s%29-to-another-pull-request%28s%29 to have more information about the procedure

Reply:
@PieterPaulussen You're welcome :)